bokehjs/src/lib/core/util/affine.ts

```diff
@@ -1,12 +1,13 @@
 import {Arrayable} from "../types"
+import {type XY} from "./bbox"
```

Comment:
It would be OK there, if fact it's unnecessary to indicate `type` import here. Type imports, so either `import {type XY}` or `import type {XY}` (same thing different granularity) are useful or even required if you have a type and an object (e.g. a class is an object) of the same name. Then you either can indicate that an import is for type purpose only, or you have to indicate if otherwise there would be a risk of introducing a value import. This is important, because type imports are erased, but value imports are not. Thus to avoid cyclic dependencies on the value-level or break JS bundle separation, it may be required to use type imports. A good example are glyphs and GL glyphs, where GL glyphs have to use type imports to not allow accidental cyclic dependency on glyphs (except for types), where glyphs already depend on GL glyphs through dynamic value imports.

Additionally, type re-exports (`export {type XY}`) are also necessary whenever re-exporting a name associated solely with a type, since recent switch to isolated modules in TypeScript. Though this is something you will rarely encounter on your own, and I'm trying to reduce unnecessary re-exports.